src/RokuDeploy.ts

```diff
@@ -237,9 +235,10 @@ export class RokuDeploy {
      * @param options
      */
     public async sideload(options: SideloadOptions): Promise<{ message: string; results: any }> {
+        this.checkRequiredOptions(options, ['host', 'password']);
         options = this.getOptions(options) as any;
         if (!options.host) {
-            throw new errors.MissingRequiredOptionError('must specify the host for the Roku device');
+            throw new Error('must specify the host for the Roku device');
```

Comment:
This is handled by checkRequiredOptions, right? I see several other places in this file that do this check. I think those can all be removed in favor of checkRequiredOptions.

Reply:
Oh oops I meant to do this. Will remove

Reply:
Fixed